### DISCUSSION TITLE
[587430]I Expect You To Die

### DISCUSSION THREAD

**Discussion by jizing** [pjy612/SteamManifestCache#851] [Share]
### Steam 游戏ID

587430

### 密钥列表

```hash
587431;eeda000e949e2a0be67b4c7e0fc40947a07bc2221b850d5bce357667ee8089e6
```

### 清单附件

[587431_5938002367866234945.zip](https://github.com/user-attachments/files/17178764/587431_5938002367866234945.zip)

### 备注

jizing@hotmail.com

**Comment by pjy612** [comment]
* 游戏ID错误

**Reply by jizing** [reply]
写成depot ID了，改了
